app/templates/courses/activate-licenses-modal.jade

```diff
@@ -63,7 +63,7 @@ block modal-body-content
         .radio.prepaid-type-line
           label
             input(type="radio", disabled=false, checked=selected, name="licenseType", value=key)
-            span=key
+            span!=key
```

Comment:
It's a little strange to use a `<span>` that we then put the HTML for a `<br>` inside. If it works, it works, but if you wanted, it might be a bit cleaner to actually put separate elements for the license description and the timeframe in the template.